### DISCUSSION TITLE
Your thoughts on existing applications

### DISCUSSION THREAD

**Discussion by gwbres** [nav-solutions/rinex#214] [Ideas]
This topic is the place to share your thoughts 💭  and ideas 💡 on the existing applications, mainly `rinex-cli` to this day.  

The main points we're interesting in getting feedback are 
- Does the current behavior and capabilities fit your requirements ? 
 - What features is this ecosystem lacking to answer your requirement
 
 `rinex-cli` was originally developped to process one RINEX file, but has evolved to answer GNSS processing better, including position solving. But GNSS applications vary a lot, and some may have different end goals that the current tool is incapable to answer

**Comment by larsnaesbye** [comment]
I have mostly been involved with supplying RINEX files to the general public. That process involved conversion from receiver-dependent binaries, 'thinning' out the frequency (from 1 sec to 5 or 30 sec intervals IIRC) and rewriting a few header fields in the RINEX files. In parallel, doing QC to evaluate the quality of the reference station data.
This has typically been through a hodgepodge of utilities, all of them closed source, some of them paid, none of them could do RINEX 4. Senior people around me pined after a `teqc` replacement.
`rinex-cli` seems to me to be able to offer that :-)
